icons/overleaf.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Overleaf icon</title><path d="M7.362 6.029a11.685 11.685 0 00-5.694 9.606 8.368 8.368 0 1011.355-7.823 7.724 7.724 0 00-2.97-.562 12.346 12.346 0 00-4.21 4.87 5.496 5.496 0 114.193 9.027 5.496 5.496 0 01-4.192-1.93 6.602 6.602 0 01-1.585-5.563c1.073-6.42 8.764-10.117 14.458-11.553a59.17 59.17 0 00-7.592 4.374c6.883 2.657 7.989-3.136 11.207-5.728C19.13-.507 7.379-.969 7.362 6.03z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Overleaf icon</title><path d="m7.3606 6.0269a11.689 11.689 0 00-5.6957 9.6089 8.3705 8.3705 0 1011.358-7.8254 7.7263 7.7263 0 00-2.9709-.56216 12.35 12.35 0 00-4.2113 4.8715 5.4976 5.4976 0 114.1943 9.0297 5.4976 5.4976 0 01-4.1933-1.9306 6.604 6.604 0 01-1.5855-5.5647c1.0733-6.4219 8.7666-10.12 14.462-11.556a59.188 59.188 0 00-7.5943 4.3753c6.8851 2.6578 7.9914-3.1369 11.21-5.7297-3.203-1.2544-14.958-1.7165-14.974 5.2846z"/></svg>
```

Comment:
Slight differences between PR (red) and source (black) as well as currently 4 decimals.
Would you be able to do a rebuild from source, exporting at 3 decimals please? 😅

![image](https://user-images.githubusercontent.com/1830867/102927907-c15c5d00-448f-11eb-887b-344e773b4694.png)

Reply:
@adamrusted Done. But a precision of 3 didn't work, only 4 did.